pkgs/development/python-modules/hiredis/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv
+, fetchPypi
+, buildPythonPackage
+, python
+}:
+buildPythonPackage rec {
+
+  pname = "hiredis";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "e97c953f08729900a5e740f1760305434d62db9f281ac351108d6c4b5bf51795";
+  };
+  
+  checkPhase = "${python.interpreter} test.py";
+
+  meta = with stdenv.lib; {
+    homepage = "http://github.com/redis/hiredis-py";
+    license = licenses.bsdOriginal;
```

Comment:
This is the three clause https://github.com/redis/hiredis-py/blob/master/COPYING